pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -76,7 +76,7 @@ let
           ++ lib.optional (cfg.enableTridactylNative or false) tridactyl-native
           ++ lib.optional (cfg.enableGnomeExtensions or false) chrome-gnome-shell
           ++ lib.optional (cfg.enableUgetIntegrator or false) uget-integrator
-          ++ lib.optional (cfg.enablePlasmaBrowserIntegration or false) plasma5.plasma-browser-integration
+          ++ lib.optional (cfg.enablePlasmaBrowserIntegration or false) plasma5Packages.plasma-browser-integration
```

Comment:
This isn't directly related to this pull request, but would it be better to import `plasma-browser-integration` directly, and `inherit` it from `plasma5Packages` in `all-packages.nix`?